### DISCUSSION TITLE
How to install Ubuntu on reTerminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#643] [Q&A]
# How to install Ubuntu on reTerminal | Seeed Studio Wiki

reTerminal-FAQ

https://wiki.seeedstudio.com/install-ubuntu-on-reterminal/

**Comment by kyle-inv-disco** [comment]
Thank you.

For those running Ubuntu 32bit - for reasons I won't speculate - sed or vim script1 and change arm64 to armfs.

**Comment by j-broome** [comment]
Unable to get this to work - tried several times and after rebooting the 2nd time, the sys light stays on and the screen is blank. Any help would be appreciated, and I can work with someone internally to get this documentation up to par.

**Reply by MatthewJeffson** [reply]
Hi, we have tested it and it seems work properly. What version did you install before
Matthew
